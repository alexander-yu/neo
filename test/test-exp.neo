void main(){ 
    var int x = 2;
    var int y = 3;
    print(x^y);
   
    print("\n");
    
    var float a = 1.5;
    var float b = 2.5;
    print(a^b);
}
