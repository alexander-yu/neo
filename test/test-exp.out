8
2.75568
